src/MerchantFrontend/src/environments/environment.prod.ts

```diff
@@ -2,6 +2,6 @@ export const environment = {
   production: true,
   hubspotPortalId: '6506294',
   hubspotFormId: 'e2d4a12e-a1c0-4ac0-87b5-ed89c490ecff',
-  apiUrl: '',
-  accessKey: ''
+  apiUrl: 'https://api.downtowns.io',
```

Comment:
Just FYI: While this works for now, this domain will be changed again in the future. Just remember to change it again then :-)

Reply:
Who changes that, should change that here too 😉